rollup/circuitcapacitychecker/impl.go

```diff
@@ -141,3 +141,31 @@ func (ccc *CircuitCapacityChecker) ApplyBlock(traces *types.BlockTrace) (*types.
 	}
 	return (*types.RowConsumption)(&result.AccRowUsage.RowUsageDetails), nil
 }
+
+func (ccc *CircuitCapacityChecker) GetTxNum() (uint64, error) {
+	ccc.Lock()
+	defer ccc.Unlock()
+
+	log.Debug("ccc get_tx_num start", "id", ccc.ID)
+	rawResult := C.get_tx_num(C.uint64_t(ccc.ID))
+	defer func() {
+		C.free(unsafe.Pointer(rawResult))
+	}()
+	log.Debug("ccc get_tx_num end", "id", ccc.ID)
+
+	result := &WrappedTxNum{}
+	if err = json.Unmarshal([]byte(C.GoString(rawResult)), result); err != nil {
+		log.Error("fail to json unmarshal get_tx_num result", "id", ccc.ID, "err", err)
+		return nil, ErrUnknown
+	}
+	if result.Error != "" {
+		log.Error("fail to get_tx_num in CircuitCapacityChecker", "id", ccc.ID, "err", result.Error)
+		return nil, ErrUnknown
+	}
+	if result.TxNum == nil {
```

Comment:
`TxNum` is `uint64`, it cannot be `nil`

Reply:
Delete this check and update `TxNum` to type uint64 in commit [6387d3f](https://github.com/scroll-tech/go-ethereum/pull/505/commits/6387d3f2b22e0928ff7ab62e53742ab687457747).